src/main/java/bio/terra/cli/apps/AppsRunner.java

```diff
@@ -36,23 +16,22 @@
  * This class runs client-side tools and manipulates the tools-related properties of the global
  * context object.
  */
-public class DockerAppsRunner {
-  private static final Logger logger = LoggerFactory.getLogger(DockerAppsRunner.class);
+public class AppsRunner {
```

Comment:
nit: I'm not quite following the "AppsRunner" name. This class seems all about creating the mappings and bindings from the user directories, env vars, workspace context, etc. to the docker environment. I think it's the `runToolCommand` is what's "running the apps", so tools = apps? Not sure what's a better name. I kind of want to focus on the "docker" part instead of what "tool" might be used in the container. brainstorming, `DockerRunner` `DockerCommandRunner` `DockerBinding` idk
No action required, pick your favorite naming.

Reply:
Good idea. I like the `DockerCommandRunner` suggestion.